docs/ABI/Mangling.rst

```diff
@@ -232,7 +232,7 @@ Entities
   ADDRESSOR-KIND ::= 'u'                     // unsafe addressor (no owner)
   ADDRESSOR-KIND ::= 'O'                     // owning addressor (non-native owner)
   ADDRESSOR-KIND ::= 'o'                     // owning addressor (native owner)
-  ADDRESSOR-KIND ::= 'p'                     // pinning addressor (native owner)
+  ADDRESSOR-KIND ::= 'p'                     // pinning addressor (native owner), not used anymore
```

Comment:
Since addressors never had public symbols do we still need to document and demangle them?

Reply:
I think we don't need to. On the other hand it does not harm to be able to demangle Swift < 5 symbols, even if they are not public.

Reply:
At the very least you can remove mangling from the ASTMangler, even if you intend on keeping the demangler and remangler support.

Reply:
I did